### ISSUE TITLE
Timer Errors

### ISSUE THREAD

**Issue by twdeleeuw** [bitfocus/companion-module-lyntec-rpc-breaker#3]
When adding the LynTec instance, I receive 3 recurring errors every 2 seconds:

04. 10:20:10 instance(LynTec): Stopping zoneCheckTimer
04. 10:20:10 instance(LynTec): Stopping transmitTimer
04. 10:20:10 instance(LynTec): Stopping zoneCheckTimer

Hardware version  2.0
Firmware Version 2.74b
Web Version 2.7x
Telnet server 1.01

I also receive this Java error upon launching the Companion App

Uncaught Exception:
TypeError [ERR_INVALID_URL]: Invalid URL: ws://192.168.3.49:undefined/stagedisplay
    at onParseError (internal/url.js:219:17)
    at parse (internal/url.js:228:3)
    at new URL (internal/url.js:311:5)
    at WebSocket.initAsClient (/Applications/Companion.app/Contents/Resources/app.asar/node_modules/ws/lib/websocket.js:470:27)
    at new WebSocket (/Applications/Companion.app/Contents/Resources/app.asar/node_modules/ws/lib/websocket.js:62:20)
    at instance.connectToProPresenterSD (/Applications/Companion.app/Contents/Resources/app.asar/lib/module/renewedvision-propresenter/index.js:668:18)
    at instance.init (/Applications/Companion.app/Contents/Resources/app.asar/lib/module/renewedvision-propresenter/index.js:160:8)
    at instance._init (/Applications/Companion.app/Contents/Resources/app.asar/lib/instance_skel.js:103:8)
    at /Applications/Companion.app/Contents/Resources/app.asar/lib/instance.js:273:23
    at EventEmitter.<anonymous> (/Applications/Companion.app/Contents/Resources/app.asar/lib/instance.js:284:6)

Running Mac OS X Sierra

**Comment by murphytodd4** [comment]
For your first issue is it causing any issues with control? Looking at the code it looks like I should have put the debug output inside of the if statement. Basically if you tell it to stop the timer and the timer is not currently running it is still sending out the info statement saying "

**Comment by murphytodd4** [comment]
Is it causing an issue with connecting and controlling the panel? If you are able to control the panel and get feedback you should be fine.

I believe what you are seeing is the fact that when the stop timer functions are called it sends out `self.log('info', "Stopping zoneCheckTimer")` and `Stopping transmitTimer")` even though the timer is not already running. The timer stop function is called every time the timer is started or restarted out of caution. I have change it so it will only send out the log notice when the timer is already running and is actually stopped but it will not be integrated into a full release till the core development team is ready.

Your error on loading has to do with the Pro Presenter module and not the LynTec module.

**Comment by twdeleeuw** [comment]
Sorry it's been so long. I got pulled away from this project a couple months ago and I now have time to revisit it. I am only given the option of Breaker 1 or Zone 1. I have no control of Breaker 1 or Zone 1 from my Stream Deck or the Companion interface. An error is listed next to the instance and there is an alarm light in the top right corner of the Stream Deck button I have programed for Zone 1.
